network/update-proxy-configs

```diff
@@ -119,21 +119,26 @@ fi
 # Pacman (archlinux) also
 if [ -d /etc/pacman.d ]; then
     if [ -n "$PROXY_ADDR" ]; then
-        cat > /etc/pacman.d/01-qubes-proxy.conf <<EOF
+        mkdir -p /usr/lib/qubes/bin
+        cat > /usr/lib/qubes/bin/pacman <<EOF
```

Comment:
If going with a non-standard path anyway, can we please not dynamically modify content of `/usr`? Better put it in `/var` or even `/run`